spec/requests/partners_requests_spec.rb

```diff
@@ -285,7 +285,7 @@
   end
 
   describe "POST #invite_partner_user" do
-    subject { -> { post invite_partner_user_partner_path(default_params.merge(id: partner.id, partner: partner.id, email: email)) } }
+    subject { -> { post invite_partner_user_partner_path(default_params.merge(id: partner.id, partner: partner.id, email: email, name: partner.name)) } }
```

Comment:
Instead of `partner.name`, make a `let(:name) { Faker::Name.unique.name }` like we do with `email` below, and then use that.

Reply:
```suggestion
    subject { -> { post invite_partner_user_partner_path(default_params.merge(id: partner.id, partner: partner.id, email: email, name: name)) } }
```